ipatests/test_integration/test_ntp_replacement.py

```diff
@@ -0,0 +1,151 @@
+#
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+#
+
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.pytest_ipa.integration import tasks
+from ipaplatform.paths import paths
+
+
+class TestNTPreplacement(IntegrationTest):
+    """
+    Test NTP Replacement
+    """
+    num_clients = 1
+
+    @classmethod
+    def install(cls, mh):
+        cls.client = cls.clients[0]
+
+    def _install_client(self, extraargs=[]):
```

Comment:
* you don't have to prefix the method with a ``_``
* don't use mutable types like list as default args. You can use ``*args`` instead and pass in extra arguments as positional arguments.

Reply:
Thanks for the suggestion, I have replaced with *args